frontend/src/Components/PageLayout/HeaderFiles/UserProductFiles/ProductsDetailsPage/ProductReviews.js

```diff
@@ -0,0 +1,37 @@
+import { Box, Typography, TextField } from "@mui/material";
+
+export default function ProductReviews({ product }) {
+  return (
+    <Box sx={{ marginTop: 2 }}>
+      <Typography variant="h6">Rating: {product.rating}</Typography>
+      <Typography variant="h6" sx={{ marginTop: 2 }}>
+        Reviews
+      </Typography>
+
+      {product.reviews && product.reviews.length > 0 ? (
+        product.reviews.map((review, index) => (
+          <Box
+            key={index}
+            sx={{ marginTop: 1, padding: 1, borderBottom: "1px solid #ddd" }}
+          >
+            <Typography variant="body1">
+              <strong>{review.reviewerName}</strong>
```

Comment:
```suggestion
              <strong>{reviewerName}</strong>
```

As we are doing Destructuring of review at the function above line 12 no need to review. we can directly call rating,reviewerName,comment